src/main/steps/children/social-worker/content.test.ts

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable jest/expect-expect */
+import { FormContent, FormFields } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { CommonContent, generatePageContent } from '../../common/common.content';
+
+import { generateContent } from './content';
+
+jest.mock('../../../app/form/validation');
+
+const CY = 'cy';
+const EN = 'en';
+
+const enContent = {
+  section: 'Your adoption agency or local authority details',
+  title: "Details about the child's social worker",
+  line1: 'You can get these details from your adoption agency or local authority.',
+  socialWorkerNameLabel: "Social worker's name",
```

Comment:
we are not specifying Label explicitly
could you please make it simple
```suggestion
  socialWorkerName: "Social worker's name",
```

This applies to below attributes as well